pkgs/development/python-modules/mcuuid/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "mcuuid";
+  version = "1.1.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "clerie";
+    repo = "mcuuid";
+    rev = "refs/tags/${version}";
+    hash = "sha256-YwM7CdZVXpUXKXUzFL3AtoDhekLDIvZ/q8taLsHihNk=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
```

```suggestion
  ];
  
  # upstream has no tests
  doCheck = false;
```
Would you please add `doCheck = false;` explicitly?
> By default the command python setup.py test is run as part of the checkPhase

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#how-to-provide-automatic-tests-to-python-packages-automatic-tests

---

done